docs/recipes/browserify.md

```diff
@@ -7,7 +7,7 @@ With this setup you can import modules into your `main.js`. Only code used in `m
 
 ### 1. Install the required plugins
 
-```
+```console
```

Comment:
`console` is not a recognized language, AFAIK. Where you thinking of `sh`/`bash`?
```suggestion
```sh
```